lib/oli/delivery/adaptive_iframe.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Oli.Delivery.AdaptiveIFrame do
```

Comment:
This module probably should be someplace under `OliWeb`, instead of `Oli` so that we do not introduce a dependency on `OliWeb.Router.Helpers` into the core `Oli` space.